helmfile/values/gatekeeper.yaml.gotmpl

```diff
@@ -0,0 +1,7 @@
+auditInterval: {{ .Values.opa.auditInterval }}
```

Comment:
Not something you need to do in this PR but we should consider using `gatekeeper` instead of `opa` imo.
```suggestion
auditInterval: {{ .Values.gatekeeper.auditInterval }}
```

Reply:
Yeah I also thought about that, but I will skip it this time.